libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/PrettyCore.hs

```diff
@@ -1712,7 +1728,7 @@ pcProposalsSnapshot :: ProposalsSnapshot era -> PDoc
 pcProposalsSnapshot x = ppSexp "ProposalsSnapShot" (map pcGovActionState (toList (snapshotActions x)))
 
 pcGovActionState :: GovActionState era -> PDoc
-pcGovActionState gas@(GovActionState _ _ _ _ _ _ _ _ _) =
+pcGovActionState gas@(GovActionState _ _ _ _ _ _ _ _ _ _) =
```

Comment:
Same here. I wonder why are we duplicating all stuff in `pretty` package and here as well.

Reply:
I had a discussion with @TimSheard and learned that the functions prefixed `pc` are different in that they are for debugging focussed pretty-printing.